controllers/apply-next/form-router.js

```diff
@@ -147,6 +142,34 @@ function initFormRouter(formModel) {
         }
     });
 
+    // Allow an application owned by this user to be deleted
+    router
+        .route('/delete/:applicationId')
+        .get((req, res) => {
+            if (req.params.applicationId && req.user.userData.id) {
+                res.render(path.resolve(__dirname, './views/delete'), {
+                    title: res.locals.formTitle,
+                    csrfToken: req.csrfToken()
+                });
+            } else {
+                res.redirect(req.baseUrl);
+            }
+        })
+        .post(async (req, res) => {
+            const deleteApplication = applicationsService.deleteApplication(
```

Comment:
How come you're not using `async/await` here?

Reply:
Good catch!